packages/gatsby/lib/query-runner/page-query-runner.js

```diff
@@ -0,0 +1,120 @@
+/***
+ * Jobs of this module
+ * - Ensure on bootstrap that all invalid page queries are run and report
+ *   when this is done
+ * - Watch for when a page's query is invalidated and re-run it.
+ ***/
+
+const _ = require("lodash")
```

Comment:
Same fact from lodash, we have four import patterns:
1. `const _ = require("lodash")`
1. `const {get, forEach} = require("lodash")`
1. `import _ from "lodash"`
1. `import {get, forEach} from "lodash"`